src/components/Description/index.tsx

```diff
@@ -1,56 +1,70 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
-import PropTypes from 'prop-types'
-import { WeakValidationMap } from 'react'
-import Box, { BoxProps } from '../Box'
 
-const Term = styled(Box.withComponent('dt'))`
+const Term = styled.dt`
   font-weight: 500;
-  color: ${({ theme }) => theme.colors.neutral.textHover};
+  color: ${({ theme }) => theme.colors.neutral.textStrong};
+  display: inline-block;
   &:after {
     content: ':';
   }
 `
-const Desc = styled(Box.withComponent('dd'))`
+
+const Desc = styled('dd', {
+  shouldForwardProp: prop => !['capitalize'].includes(prop),
+})<{ capitalize?: boolean }>`
+  display: flex;
+  align-items: center;
   color: ${({ theme }) => theme.colors.neutral.text};
-  margin: 0;
-  margin-top: ${({ theme }) => theme.space['1']};
+  margin: 8px 0 0 0;
+  ${({ capitalize }) =>
+    capitalize &&
+    css`
+      text-transform: capitalize;
+    `}
 `
 
-const StyledBox = styled(Box, {
-  shouldForwardProp: prop => !['inline', 'selectable'].includes(prop),
-})<{ inline: boolean; selectable: boolean }>`
+interface DescriptionProps {
+  ellipsis?: boolean
+  inline?: boolean
+  userSelect?: boolean
+  width?: string
+}
+
+const BareDescription = styled('dl', {
+  shouldForwardProp: prop =>
+    !['inline', 'ellipsis', 'userSelect'].includes(prop),
+})<DescriptionProps>`
   font-size: 16px;
   line-height: 16px;
   margin: 0;
 
-  ${Desc} + ${Term} {
-    margin-top: ${({ theme }) => theme.space['2']};
-  }
-
-  ${({ theme, inline }) =>
-    inline
-      ? css`
-          & > ${Term} {
-            float: left;
-            clear: left;
-            margin-right: ${theme.space['1']};
-          }
-
-          & > ${Desc} {
-            margin-top: 0;
-          }
+  ${({ inline }) =>
+    inline &&
+    css`
+      display: flex;
+      justify-content: flex-start;
+      align-items: center;
+      ${Desc} {
+        margin: 0 0 0 8px;
+      }
+    `};
 
-          ${Desc} + ${Term} + ${Desc} {
-            margin-top: ${theme.space['2']};
-          }
-        `
-      : ``}
+  ${({ ellipsis, width }) =>
+    ellipsis &&
+    css`
+      ${Desc} {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        width: ${width};
+      }
+    `}
 
-  ${({ theme, selectable }) =>
-    selectable &&
+  ${({ userSelect, theme }) =>
+    userSelect &&
```

Comment:
Use ternary instead ?

```ts
ellipsis ? ... : null
```

Reply:
@matthprost Why ? In template literal it does not makes a lot of sense 🤔

Reply:
@chambo-e why not?

Reply:
Sure, it's just more verbose with no safety addition :)